python-avd/pyavd/_eos_designs/structured_config/base/snmp_server.py

```diff
@@ -191,9 +191,13 @@ def _snmp_local_interfaces(self: AvdStructuredConfigBaseProtocol, source_interfa
         if not source_interfaces_inputs:
             # Empty dict or None
             return
-        local_interfaces = self._build_source_interfaces(source_interfaces_inputs.mgmt_interface, source_interfaces_inputs.inband_mgmt_interface, "SNMP")
-        for local_interface in local_interfaces:
-            self.structured_config.snmp_server.local_interfaces.append(EosCliConfigGen.SnmpServer.LocalInterfacesItem(**local_interface))
+        local_interfaces = self._build_source_interfaces(
+            source_interfaces_inputs.mgmt_interface,
+            source_interfaces_inputs.inband_mgmt_interface,
+            "SNMP",
```

Comment:
just for readability
```suggestion
            error_context="SNMP",
```

Reply:
Done.